api/authFunctions.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { logout} from './authFunctions';
+
+//logout method
+
+describe('Auth Functions', () => {
+  // Test the login function
+  describe('login', () => {
```

Comment:
We have formatting issues here. Is ESLint and Prettier installed in your IDE?

Reply:
Where I have to install it globally, correct?

Reply:
Just installed it globally into my IDE so hopefully things look prettier now :)

Reply:
Format is still way off. Did you follow @ryandotfurrer's doc to add it to your IDE?

Reply:
@shashilo I modified the `prettier.config.js` file accordingly to what Ryan stated in group